Makefile

```diff
@@ -19,6 +19,7 @@ target = kuma
 requirements = -r requirements/local.txt
 # set Django settings module if not already set as env var
 export DJANGO_SETTINGS_MODULE ?= kuma.settings.testing
+export UID ?= $(shell id -u)
```

Comment:
Not in a Makefile. See https://www.gnu.org/software/make/manual/make.html#Shell-Function for more details.